odl/operator/oputils.py

```diff
@@ -319,6 +319,76 @@ def rmatvec(v):
                                               dtype=dtype)
 
 
+def as_scipy_functional(func, return_gradient=False):
+    """Wrap ``op`` as a function operating on linear arrays.
+
+    This is intended to be used with the scipy solvers.
+
+    Parameters
+    ----------
+    func : `Functional`
+        A functional that should be wrapped
+    return_gradient : bool, optional
+        If the gradient of the functional should also be returned
+
+    Returns
+    -------
+    function : ``callable``
+        The wrapped functional.
+    gradient : ``callable``, optional
```

Comment:
Well the user has an option (specifying a parameter), but i see what you mean.

Since I can't come up with anything smarter, I'll go ahead for now.